Respond to @orchestrator mention in #205

This task was created from an @orchestrator mention.

- **Repo:** `gabrielkoerich/orchestrator`
- **Target:** #205
- **Author:** @gabrielkoerich
- **Comment:** https://github.com/gabrielkoerich/orchestrator/issues/205#issuecomment-3940290727
- **Created:** 2026-02-22T06:37:28Z

### Mention Body

```markdown
Follow-up on the @orchestrator mention in PR #183:

- Reproduced the CI failures: `shellcheck` + `semgrep-and-secrets` were tripping over backticks inside double-quoted owner-command ack messages in `scripts/lib.sh` (command substitution + SC1072/SC1073 around `/context <text>`).
- Fixed quoting by switching those ack strings to safer patterns (single quotes / `printf` / `$'...'` where needed) so backticks are treated as literals.
- Ran `bats tests/orchestrator.bats` locally (pass).

Fix is up in PR #206:
- https://github.com/gabrielkoerich/orchestrator/pull/206

Next steps:
- Review + merge #206 once CI is green.
```

### Instructions

Respond back on #205 with your results and next steps.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -28,6 +28,13 @@ bats tests/orchestrator.bats   # main test file
 # bats tests                  # run all tests
 ```
 
+## Shell scripting & CI linting
+
+CI runs `shellcheck` and `semgrep-and-secrets` on `scripts/*.sh`.
+
+- Prefer `$(...)` over backticks for command substitution.
+- If you need to include *literal* backticks in user-facing strings (help/ack messages), avoid putting them in a double-quoted shell string (they will be command-substituted). Use single quotes, a quoted heredoc, or `printf '%s'`.
+
 ## Commit message conventions
 
 Use Conventional Commits:
@@ -48,4 +55,3 @@ git checkout -b my-branch origin/main
 git commit -m "docs: add contributing guide"
 git push -u origin my-branch
 ```
-

